app/Models/User.php

```diff
@@ -14,6 +15,7 @@
 use Illuminate\Foundation\Auth\User as Authenticatable;
 use Illuminate\Notifications\DatabaseNotification;
 use Illuminate\Notifications\Notifiable;
+use Illuminate\Support\Facades\Cache;
```

Comment:
Siehe Suggestion unten..

```suggestion
use Illuminate\Support\Facades\RateLimiter;
```